app/models/user.rb

```diff
@@ -18,8 +18,8 @@ class User < ApplicationRecord
   attr_writer :login
 
   # Only allow letter, number, underscore and punctuation.
-  validates_format_of :username, with: /^[a-zA-Z0-9_\.]*$/, :multiline => true
-  validates :username, presence: true, uniqueness: { case_sensitive: false }
+  validates_format_of :username, with: /^[a-zA-Z0-9_\.]*$/, multiline: true
+  validates :username, :email, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
Yo personalmente prefiero que hagas un validate por campo.
```suggestion
  validates :username, presence: true, uniqueness: { case_sensitive: false }
  validates :email, presence: true, uniqueness: { case_sensitive: false }
```